utilities/repostatus.py

```diff
@@ -0,0 +1,50 @@
+import os
```

Comment:
2 things for this file.
1. Could you wrap the work up into a function so testing doesn't run it each time? Something like this:
```
if __name__ == "__main__":
    generate_html()
```
2. Could you add some documentation about how to run it and what it does.